src/components/Comments.jsx

```diff
@@ -0,0 +1,43 @@
+import { getComments } from '../utils/api';
+import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import moment from 'moment';
```

Comment:
From my understanding, moment.js is deprecated. This doesn't necessarily mean that it won't work, but it does mean that it is not being maintained. There could well be a 'moment'. when it stops working on newer versions of frameworks and runtime environments. Just something to be aware of :)